SourceX/DiabloUI/fonts.h

```diff
@@ -7,7 +7,11 @@
 #include "DiabloUI/art.h"
 
 #ifndef TTF_FONT_PATH
+#ifndef __AMIGA__

```

Comment:
This should be set via CMake.

Reply:
Had some issues getting it to compile when adding it in CMakeList.txt so put it here before going to bed :/ Should I add it to the CLI instead of txt file?

Reply:
It should work just fine in CMakeLists.txt, must be set before the loop that adds defines.

Reply:
Done